signing/transaction-manager.js

```diff
@@ -138,3 +138,27 @@ export async function pollTxInclusion(txHash, iteration = 0) {
     )
   }
 }
+
+// limitation of the Ledger Nano S, so we pick the top 5 rewards and inform the user.
```

Comment:
can we put this in another file? doesn'T really belong in the transaction manager

Reply:
Done